src/browser/atom-window.coffee

```diff
@@ -73,6 +73,9 @@ class AtomWindow
     @browserWindow.loadUrl @getUrl(loadSettings)
     @browserWindow.focusOnWebView() if @isSpec
 
+    if locationsToOpen.length is 1 and locationsToOpen[0].pathToOpen is null
```

Comment:
Just curious, I feel like I've seen `is null` being used lately, is there a reason to use it instead of just a `?` ?